### ISSUE TITLE
Woah there, that's a little fast!

### ISSUE THREAD

**Issue by UnKn0wnC0d3s** [jamieweavis/rocket-league-garage-trade-bumper#5]
So RL Garage is now sending an error when using the "Bump All" option, they probably patched the ability to bump them all at once, now just one of them is bumped, maybe this could be solved with a 1-2 second interval while bumping each trade

**Comment by UnKn0wnC0d3s** [comment]
![image](https://user-images.githubusercontent.com/66189027/108640809-915e1580-7450-11eb-9d9b-7ff1efe0a73c.png)

Request:
```
Request URL: https://rocket-league.com/ajaxfunctions/bumpTrade.php
Request Method: POST
Status Code: 200 
```
Response:
`Woah there, that's a little fast!`

**Comment by jamieweavis** [comment]
Thanks for the report @UnKn0wnC0d3s - I'll look into this soon!

**Comment by iTechsTR** [comment]
Yeah, was just coming here to report as well. Even if I try to bump them manually, it doesn't allow me to, if like 2 seconds hasn't passed.

**Comment by jamieweavis** [comment]
A fix for this is now available in `v0.4.1` by adding a 2 second stagger between each bump when bumping all trades

This version has been submitted to the Chrome Web Store but it usually takes several days to be approved, if you want to use the latest version early you can download and install it manually with [these instructions](https://github.com/jamieweavis/rocket-league-garage-trade-bumper#manual)

**Comment by jamieweavis** [comment]
It looks like this was approved really quickly this time, it's already available on the Chrome Web Store 🎉
